### ISSUE TITLE
Support for Terraform 0.12.x

### ISSUE THREAD

**Issue by dgmora** [spotinst/terraform-spotinst-modules#17]
Terraform version: 0.12.12

I stumbled upon the error in https://github.com/spotinst/terraform-spotinst-modules/issues/16.

This error is happening because the syntax used is not suitable for 0.12. I fixed it locally in https://github.com/dgmora/terraform-spotinst-modules/

So having this made it work for me
```hcl
module "spotinst_ocean_controller" {
  source = "github.com/dgmora/terraform-spotinst-modules//spotinst_ocean_controller"

  . . . # all the things
}
```

I did not submit a PR because the fork only fixes one of the modules, but I can do so if you want to.

**Comment by JulianGindi** [comment]
I would submit your change as a PR. I ran into the same issue and changed it in a personal fork. Your fixes worked for me. I think it's fine if it's a single module, since not everyone uses all modules. Let me know if I can help in any way.

**Comment by liranp** [comment]
Thank you for letting us know. Fixed by #18 and #20. Closing for now.
